blivet/formats/fs.py

```diff
@@ -251,6 +253,14 @@ def label_format_ok(self, label):
     label = property(lambda s: s._get_label(), lambda s, l: s._set_label(l),
                      doc="this filesystem's label")
 
+    def can_assign_uuid(self):
```

Comment:
This would have sounded odd for `can_reassign_uuid`: `can_reset_uuid` sounds a bit like "use the autogenerated UUID".